backend/src/graphql/resolver/TransactionResolver.test.ts

```diff
@@ -0,0 +1,348 @@
+/* eslint-disable @typescript-eslint/no-explicit-any */
+/* eslint-disable @typescript-eslint/explicit-module-boundary-types */
+
+import { EventProtocolType } from '@/event/EventProtocolType'
+import { userFactory } from '@/seeds/factory/user'
+import {
+  confirmContribution,
+  createContribution,
+  login,
+  sendCoins,
+} from '@/seeds/graphql/mutations'
+import { bobBaumeister } from '@/seeds/users/bob-baumeister'
+import { garrickOllivander } from '@/seeds/users/garrick-ollivander'
+import { peterLustig } from '@/seeds/users/peter-lustig'
+import { stephenHawking } from '@/seeds/users/stephen-hawking'
+import { EventProtocol } from '@entity/EventProtocol'
+import { Transaction } from '@entity/Transaction'
+import { User } from '@entity/User'
+import { cleanDB, testEnvironment } from '@test/helpers'
+import { logger } from '@test/testSetup'
+import { GraphQLError } from 'graphql'
+
+let mutate: any, query: any, con: any
+let testEnv: any
+
+beforeAll(async () => {
+  testEnv = await testEnvironment(logger)
+  mutate = testEnv.mutate
+  query = testEnv.query
+  con = testEnv.con
+  await cleanDB()
+})
+
+afterAll(async () => {
+  await cleanDB()
+  await con.close()
+})
+
+let bobData: any
+let peterData: any
+let user: User[]
+
+describe('send coins', () => {
+  beforeAll(async () => {
+    await userFactory(testEnv, peterLustig)
+    await userFactory(testEnv, bobBaumeister)
+    await userFactory(testEnv, stephenHawking)
+    await userFactory(testEnv, garrickOllivander)
+
+    bobData = {
+      email: 'bob@baumeister.de',
+      password: 'Aa12345_',
+      publisherId: 1234,
```

Comment:
Publisher Id is not needed. May be we should include the password in the user interface definition of the seed and add the password as well to the user objects. Then we do not need this helpers. We can pass the imported user objects directly to the login call. This makes this test scenarios much better to read. Would you mind doing this?

Reply:
I agree that it is not needed.
However, are you sure User objects should store their password? I don't think that is the responsability of a User object. I mean, when we are aiming at an instantiated User, we are talking about a user interacting with the app, and so already logged. For me it is DBUser handling the password (which is already implemented).
I'm not sure of this, of course I can implement what you say, but let me know if I am wrong prior to that.
Thanks!!!